### PR TITLE
myPage button now produce myPage scene

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/actions/navigatorActions.js [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+    navigator
+  }
+};
+
+export const setMyPageSceneIndex = (index) => {
+  return {
+    type: types.setMyPageSceneIndex,
+    index
+  }
+};
```
Comment (festinalente91) [inline shared/actions/navigatorActions.js]: add a new line

Comment (ochanje210) [inline shared/actions/navigatorActions.js]: k

File: shared/components/headerbox.js [hunk 2]
```diff
@@ -104,9 +113,12 @@ Headerbox.propTypes = {
...
+  setMyPageSceneIndex: PropTypes.func,
   tabview_index: PropTypes.any,
   tabview_routes: PropTypes.any,
   route: PropTypes.any,
   navigator: PropTypes.any,
-  sceneIndex: PropTypes.any
+  sceneIndex: PropTypes.any,
+  myPageSceneIndex: PropTypes.any,
+  myPageNavigator: PropTypes.any
 };
```
Comment (festinalente91) [inline shared/components/headerbox.js]: ditto

Comment (ochanje210) [inline shared/components/headerbox.js]: no it's not

Comment (ochanje210) [inline shared/components/headerbox.js]: https://66.media.tumblr.com/36f4d111796bd2063e1b707e99067e23/tumblr_oag1yjQA9x1vztgqao3_1280.png

**Comment by elegant651** [comment]
LGTM

File: shared/components/headerbox.js [hunk 3]
```diff
@@ -20,38 +20,47 @@ export default class Headerbox extends Component {
...
-    console.log("change");
-    console.log(this.props.sceneIndex);
     return this.props.onForward(
       this.props.sceneIndex,
       this.props.navigator
     )
   }
 
+  _onForwardMyPage() {
+    console.log(this.props.myPageNavigator);
```
Comment (yuda1124) [inline shared/components/headerbox.js]: I think you forgot to remove it kk

**Comment by mun625** [comment]
LGTM

**Pull request merged**